dayrepo/snippets/urls.py

```diff
@@ -11,4 +11,5 @@
     path("employee/", views.get_employee,name="get_employee"),
     path("snippet-list/", views.snippet_list,name="snippet_list"),
     path("snippet-post/", views.snippet_post,name="snippet_post"),
+    path("snippet-checklist/", views.checklist_post,name="checklist_post"),
```

Comment:
ちなみにずっと思っていて質問するの忘れてたから今更なんだが、

URLのパスで`snippets/snippet-checklist/`ってなんかおかしいかも。
snippet 2回出てきてるの違和感

```suggestion
    path("checklist/", views.checklist_post,name="checklist_post"),
```